source/lac/sparse_direct.cc

```diff
@@ -933,6 +926,9 @@ void
 SparseDirectMUMPS::initialize_matrix(const Matrix &matrix)
 {
   Assert(matrix.n() == matrix.m(), ExcMessage("Matrix needs to be square."));
+  // Here we should be checking if the matrix is respecting the symmetry given
+  //(I.E. sym = 0 for non-symmetric matrix, sym = 1 for posdef matrix, sym = 2
+  // for general symmetric).
```

Comment:
I don't understand this comment. We can't *check* whether a matrix has these properties with reasonable effort, but we are getting this information through the `AdditionalData` object now.

Reply:
That was a leftover from the previous PR. Fixed.